docs/getting_started.rst

```diff
@@ -61,16 +97,12 @@ Pauli Strings also support operations like addition, multiplication, and more. F
     P1.dim
     P1.n_qubits
 
-    # Add two Pauli strings. Return type is a PauliOp because
-    # the product is not a Pauli string
-    P3 = P1 + P2
-
     # Multiply two Pauli strings.
     phase, new_string = P1 @ P2
 
 
-We can also do more complicated things, like compute the action of a Pauli string :math:`\mathcal{\hat{P}}` on a quantum state :math:`| \psi \rangle`, :math:`\mathcal{\hat{P}}| \psi \rangle`, or
-compute the expectation value of a Pauli string with a state :math:`\langle \psi | \mathcal{\hat{P}} | \psi \rangle`:
+We can also do more complicated things, like compute the action of a Pauli string :math:`\mathcal{\hat{P}}` on a vector :math:`| \psi \rangle`, :math:`\mathcal{\hat{P}}| \psi \rangle`, or
+compute the expectation value of a Pauli string with a state :math:`\langle \psi | \mathcal{\hat{P}} | \psi \rangle`. As a side note, in this guide we will use state and vector interchangeably:
 
 .. code-block:: python
```

Comment:
It won't let me make suggestions for the whole block, so here's an edited version:
```python
import numpy as np
# Apply P to a state
P = fp.PauliString('XY')
state = np.array([1, 0, 0, 1], dtype=complex)
new_state = P.apply(state)

# Compute the expected value of P with respect to a state or a batch of states
value = P.expectation_value(state)

states = np.random.randn(4, 8) + 1j * np.random.randn(4, 8)
values = P.expectation_value(states)
```